spec/api/sul_bib/authorship_api_spec.rb

```diff
@@ -458,71 +423,51 @@ def check_response_error(code)
   # PATCH
 
   context 'PATCH /authorship' do
-    context 'success' do
-      let(:http_request) do
-        patch '/authorship', request_data.to_json, headers
-        expect(response.status).to eq(200)
-      end
+    let(:http_request) { patch '/authorship', request_data.to_json, headers }
```

Comment:
:+1: - I can see the extraction here for both success and failure, with the addition of the after block on success.  Just wondering whether the failure context could have a generic after block to test for failure (it's not there before, so this is asking for an addition that may not be necessary, it's not a blocker).

Reply:
I think the failures have various behaviors (404 vs. 401).